scripts/partition_manager.py

```diff
@@ -318,85 +360,161 @@ def parse_args():
                              "image-name:pm.yml-path:build_dir:out_dir")
     parser.add_argument("--app-pm-config-dir", required=True,
                         help="Where to store the 'pm_config.h' of the root app.")
+    parser.add_argument("--app-build-dir", required=True,
+                        help="Path to of the root app's build directory.")
+    parser.add_argument("-o", "--output", required=False, type=str,
```

Comment:
After testing this, I don't see the point of having this parameter. This path is relative to the output dir of the active image, and must be unique. As with the other two generated files, this should be hard coded in the script.

Reply:
Removed